docs/decisions/0002-footer-links.rst

```diff
@@ -0,0 +1,168 @@
+component, modify it, and then install the fork as a package instead of the
+default footer at build time of MFEs.
+
+We want to make the MFE footer to closer resemble the legacy UI footer by
+adding the same footer links as in the legacy UI, and to make it easier to
+style the footer component.
+
+Decision
+********
+
+.. This section describes our response to these forces. It is stated in full sentences, with active voice. "We will …"
+
+Links
+=====
+
+For specifying the footer links, we are going to use `MFE's config mechanism`_.
+That would allow to specify a list of links in various ways:
+
+* At build time:
+  * Via environmental variables
+  * By providing them in ``env.config.js``, like so:
+
+    .. code-block:: javascript
+
+        const config = {
+            FOOTER_LINKS: [
+                {"url": "https://google.com", "text": "Terms of service"},
+                {"url": "https://duckduckgo.com", "text": "Acceptable Use Policy"},
+                {"url": "https://openedx.org", "text": "Privacy Policy"},
+            ],
+        };
+
+        export default config;
+
+* At runtime, via ``MFE_CONFIG`` in LMS's Django settings:
+
+    .. code-block:: python
+
+        MFE_CONFIG = {
+            "FOOTER_LINKS": [
+                {"url": "https://google.com", "text": "Terms of service"},
+                {"url": "https://duckduckgo.com", "text": "Acceptable Use Policy"},
+                {"url": "https://openedx.org", "text": "Privacy Policy"},
+            ],
+        }
+
+In case the array is undefined or empty, we won't render any links (or it's
+container).
+
+.. _MFE's config mechanism: https://github.com/openedx/frontend-platform/blob/master/src/config.js
+
+Styles
+======
+
+Since the root of the footer component is rendered as a footer HTML element, it
+is unique enough to use ``footer`` as selector. To make it more convenient to
+style footer's children, we will add unique class names, which will simplify
+the SCSS selectors.
+
+These SCSS styles can be put in a custom `branding package`_ into
+``paragon/_overrides.scss``.
+
+.. _branding package: https://github.com/openedx/brand-openedx
+
+Consequences
+************
+
+.. This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+
+Positive
+========
+
+* There will be an option to make the MFE footer look similar to the legacy UI
+  footer, without having to create and maintain a fork.
+* Can add custom text/urls for links, unlike what branding v1 API allows.
+
+Negative
+========
+
+* There are already parameters that allow configuring the footer links for the
+  legacy UI, which means that configuration will be duplicated (potentially in
+  multiple places), in cases when the aim is to make the links in the legacy UI
+  and MFE footers that same.
```

Comment:
Perhaps this could build on that API, i.e. take that existing config, + add support for extending it, so that the data from existing branding config will automatically be available if that api is enabled, but so will additional links you add in the config if needed.

Reply:
@xitij2000 Added a note about it as well.